### DISCUSSION TITLE
Adding Chessboards to the environment?

### DISCUSSION THREAD

**Discussion by Doodle1106** [microsoft/AirSim#4067] [Support Q&A]
### What's your question?
I there a way to add chessboard/charuco boards to the environment?

### Include context on what you are trying to achieve
I am trying to test my camera calibration algorithms and I'd like to add some chessboards of different sizes to the environment.

**Comment by jonyMarino** [comment]
Hi @Doodle1106, and welcome!
This is an unreal question. You can use any chessboard image like
[this one](https://www.google.com/imgres?imgurl=https%3A%2F%2Fimage.shutterstock.com%2Fimage-vector%2Fabstract-background-chess-board-optical-260nw-1717522429.jpg&imgrefurl=https%3A%2F%2Fwww.shutterstock.com%2Fsearch%2Fchess%2Bboard%2Btexture&tbnid=rm72hJR2kCtdtM&vet=12ahUKEwjHsOWS-rjzAhW_BbkGHQioCDUQMyhNegQIARBp..i&docid=K3xaE0cUkRumYM&w=260&h=280&q=chess%20board%20texture%20for%20unreal%20engine%204&ved=2ahUKEwjHsOWS-rjzAhW_BbkGHQioCDUQMyhNegQIARBp) and apply it to any object in your environment as a texture, if it is on one of [this formats](https://docs.unrealengine.com/4.26/en-US/RenderingAndGraphics/Textures/Importing/)
